src/components/global/table/toolbar.tsx

```diff
@@ -18,8 +21,12 @@ const Toolbar = () => {
           disabled: false,
         }}
       />
-      <div className="cursor-pointer">REFRESH</div>
-      <div className="cursor-pointer">TRASH</div>
+      <div className="cursor-pointer">
```

Comment:
instead of using `cursor-pointer`, just make these buttons as we have to add functionality to it at some point.